LogcatCoreLib/build.gradle

```diff
@@ -16,6 +16,7 @@ android {
 }
 
 dependencies {
+    implementation "androidx.core:core-ktx:1.3.2"
```

Comment:
This is not needed and pointless

Reply:
I can however extract the code and create an extension class or inline the code, to remove the additional dependency which is used for just `one` extension function, I don't mind doing this

Reply:
Ahh, maybe my fault

Reply:
> I can however extract the code and create an extension class or inline the code, to remove the additional dependency which is used for just one extension function, I don't mind doing this

It's not necessary. Either you rebase and I merge this, or I merge #195 
Please simply tell it me

Reply:
> Ahh, maybe my fault

It is my fault !